### PR TITLE
Marius/10/navbar

### PR DESCRIPTION
**Pull request by EnvyPhD**

### CODE REVIEW

File: src/dataexplorer/dataexplorer-application/src/components/App.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 import { Route, Switch } from "react-router-dom";
-
-import Dashboard from "./Dashboard/Dashboard";
+import Navigation from './Navigation/Navigation'
 
 const App = () => (
-  <Switch>
-    <Route exact path="/" component={Dashboard} />
-  </Switch>
+  <React.Fragment>
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/App.tsx:6]:
```suggestion
  <>
```

File: src/dataexplorer/dataexplorer-application/src/components/App.tsx [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+    <Switch>
+      <Navigation />
+    </Switch>
+  </React.Fragment>
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/App.tsx:10]:
```suggestion
  </>
```

File: src/dataexplorer/dataexplorer-application/src/components/MinePasienter/MinePasienter.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Dashboard = () => {
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/MinePasienter/MinePasienter.tsx:3]:
```suggestion
const MinePasienter = () => {
```
I would prefer the component to be named the same as the file and don't have duplicated componentnames

File: src/dataexplorer/dataexplorer-application/src/components/Navigation/Navigation.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    },
+    {
+      "id": 4,
+      "url": "/Search",
+      "title": "Filtrer på egenskaper"
+    },
+  ]
+  }
+
+// for (let index = 0; index < array.length; index++) {
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/Navigation/Navigation.tsx:37]: Don't think the commented code is needed for later use. Maybe remove these comments?

File: src/dataexplorer/dataexplorer-application/src/components/Navigation/Navigation.tsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+      mode="inline" 
+      defaultSelectedKeys={['1']}>
+      
+      {data.items.map(({id, url, title}) => (
+        <MenuItem key={id}> 
+          <Link to={url}> {title} </Link> 
+        </MenuItem>
+        ))}
+
+        {/* <Menu.Item key="1">
```
Comment (froydis98) [inline src/dataexplorer/dataexplorer-application/src/components/Navigation/Navigation.tsx:73]: Same for these comments

File: src/dataexplorer/dataexplorer-application/src/components/MinePasienter/MinePasienter.tsx (hunk 3 above)
Comment (losolio) [reply src/dataexplorer/dataexplorer-application/src/components/MinePasienter/MinePasienter.tsx:3]: Should we rename `MinePasienter` to `MyPatients`?

Comment (EnvyPhD) [reply src/dataexplorer/dataexplorer-application/src/components/MinePasienter/MinePasienter.tsx:3]: Nice catch. I copied the intially empty Dashboard component to create the two others, so didnt realize this.

File: src/dataexplorer/dataexplorer-application/src/components/Navigation/Navigation.tsx (hunk 4 above)
Comment (EnvyPhD) [reply src/dataexplorer/dataexplorer-application/src/components/Navigation/Navigation.tsx:37]: Agreed

File: src/dataexplorer/dataexplorer-application/src/components/Navigation/Navigation.tsx (hunk 5 above)
Comment (EnvyPhD) [reply src/dataexplorer/dataexplorer-application/src/components/Navigation/Navigation.tsx:73]: Agreed

File: src/dataexplorer/dataexplorer-application/src/components/Dashboard/Dashboard.tsx [hunk 6]
```diff
@@ -1,7 +1,37 @@
...
+      <Row gutter={[60, 40]} justify={"center"}>
+      <Col span={300}>
+        <Search placeholder="Finn et Skjema!" enterButton />
+      </Col>
+      </Row>
+      <Row style={{paddingTop: 100}}/>
+      <Row gutter={[105, 105]} justify={"center"}>
+      <Col span={8}>
+        <Card title="Mottatte Skjema" hoverable> 
+          <p>Velkommen tilbake, Dr. House!</p>
```
Comment (losolio) [inline src/dataexplorer/dataexplorer-application/src/components/Dashboard/Dashboard.tsx:24]: 🤣

**Pull request merged**